### DISCUSSION TITLE
Dealing with different target database names - is this a bug?

### DISCUSSION THREAD

**Discussion by cboger77** [pgmodeler/pgmodeler#1562] [Q&A]
Have  model that we use with test, development, and production databases.  The problem I am having is with permissions set on the Database itself.  The statements generated on DIFF always contain the name of the database in the model, not the target.  Other aspects of the DIFF appear to work correctly.  Is there a best practice or some other advice anyone has?

**Comment by rkhaotix** [comment]
This is the expected behavior. pgModeler will always use the database model name as reference. For now there's no workaround unless you change the resulting diff code by hand. I'll register this as a feature request for future implementation.
